components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/services/TenantMgtImpl.java

```diff
@@ -317,6 +308,11 @@ public void deactivateTenant(int tenantId) throws TenantMgtException {
         }
     }
 
+    private String maskOrUnmaskContent(String content) {
```

Comment:
Shall we rename this which gives the meaning ? i.e maskContent ?

Reply:
Reanmed to `applyLogMaskingIfEnabled`